sonar/json_schemas/deposits_json_schema.py

```diff
@@ -32,14 +32,33 @@ def process(self):
         """
         schema = super().process()
 
-        if not current_user_record or current_user_record.is_moderator:
+        organisation = []
+        if current_user_record:
+            organisation = current_user_record.replace_refs() \
+                .get('organisation')
+
+        if not current_user_record or (current_user_record.is_moderator and \
+            organisation.get('isDedicated')):
             return schema
 
+        # Remove some fields on json for the shared organisation
+        if organisation.get('isShared'):
```

Comment:
Is `organisation` a list or a simple value? L. 35 suggest a list but this line suggest a simple value.

Reply:
To be safer, we can keep the fields only for dedicated:
```
if not organisation.get('isDedicated', False):
```

Reply:
Error in my code: `organisation = {}`